.prettierrc

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Empty prettier config so that defaults are used and it works with the VS Code Prettier extension. I may set up prettier tooling at some point later.